source/_components/sensor.bme680.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: "BME680 Sensor"
+description: "Instructions how to integrate a BME680 sensor into Home Assistant."
+date: 2018-01-16 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
```

Comment:
This is actually a sensor by Bosch right? Why show the Raspberry PI logo?

Reply:
I used the same logo as was used for the BME280 sensor which this is closely related to.  It is a directly connected I2C sensor typically used on platforms such as the raspberry pi as it needs a GPIO connection.  I am open to any suggestions as to what to change it to  but again, I was simply mirroring the existing BME280 component's logo in the existing documentation for consistency.

Reply:
I think for consistency with the BME280, we should keep it as the RPi logo for now.  However if anyone wanted to propose a general change later to all these RPi library based sensors, I would not have objections.

The other thing is I have no idea on the requirements to use Bosch's logo and if they would have any objections or possible legal concerns for doing so.